lib/Club.js

```diff
@@ -4,7 +4,7 @@ class Club {
     this.name             = data.name
     this.type             = data.type
     this.badgeId          = data.badgeId
-    this.description      = data.description
+    this.description      = data.description ? data.description : `null`
```

Comment:
I would suggest using `data.description || null` in this case? A string containing '_null_' is not really useful
```suggestion
    this.description      = data.description || null
```

Reply:
Alright, that seems like a good idea.